README.md

```diff
@@ -352,4 +352,22 @@ The following contribution guidelines should be followed:
 
 1. Code should be styled according to the included [style.xml](style.xml) eclipse rules.
 
-1. A pull request should be sent for review only after the github action associated with this repository is automatically executed when a pull request is created.
+2. A pull request should be sent for review only after the github action associated with this repository is automatically executed when a pull request is created.
```

Comment:
Markdown should allow for using the value `1.` through this list and it will be rendered as different values automatically. Could you please try using `1.` to replace both the values `2.` and `3.`?